modules/nf-core/jvarkit/dict2bed/meta.yml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "jvarkit_dict2xml"
+description: Extract BED file from hts files containing a dictionary (VCF,BAM, CRAM, DICT, etc...)
+keywords:
+  - vcf
+  - bcf
+  - bed
+  - dict
+  - dictionary
+  - fasta
+  - fai
+tools:
+  - "jvarkit":
+      description: "Java utilities for Bioinformatics."
+      homepage: "https://github.com/lindenb/jvarkit"
+      documentation: "https://jvarkit.readthedocs.io/"
+      tool_dev_url: "https://github.com/lindenb/jvarkit"
+      doi: "10.6084/m9.figshare.1425030"
+      licence: ["MIT License"]
+      args_id: "$args"
```

Comment:
any idea if we need to refer args2?

Reply:
no-idea, you're the boss of nf-core here :-)